app/src/main/java/com/cornellappdev/uplift/ui/components/capacityreminder/LocationsToRemind.kt

```diff
@@ -0,0 +1,101 @@
+package com.cornellappdev.uplift.ui.components.capacityreminder
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.uplift.util.GRAY03
+import com.cornellappdev.uplift.util.montserratFamily
+
+/**
+ * Section with locations to remind the user to check the capacity.
+ */
+@Composable
+fun LocationsToRemind(
+    initialSelectedGyms: Set<String> = emptySet(),
+    onGymSelected: (Set<String>) -> Unit
+) {
+    val topRow = listOf("Teagle Up", "Teagle Down", "Helen Newman")
+    val bottomRow = listOf("Toni Morrison", "Noyes")
+
+    var selectedGyms by remember { mutableStateOf(initialSelectedGyms) }
```

Comment:
Same idea here with the whole contractually obligated thing. I'd recommend for simplicity just getting rid of the state (changing `initialSelectedGyms` to `selectedGyms`) and then just trusting `selectedGyms` like you normally would.